cvat/apps/engine/models.py

```diff
@@ -49,6 +49,9 @@ class Task(models.Model):
     z_order = models.BooleanField(default=False)
     flipped = models.BooleanField(default=False)
     image_quality = models.PositiveSmallIntegerField(default=50)
+    start_frame = models.PositiveIntegerField(default=0)
```

Comment:
I don't see in your code that these values are used for `dump file` functionality. It means that any dump file will contain incorrect frame numbers. It is very critical as I wrote in the past.

Reply:
Added the 3 fields into dump file.